src/SiteExtensions/LoggingAggregate/version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.0.0</VersionPrefix>
-    <VersionSuffix>preview</VersionSuffix>
```

Comment:
cc @shirhatti

Reply:
Prefix update in necessary so when we start building site extension as part of 3.0 previews we won't get the conflicting versions.